app/views/record/view.html.erb

```diff
@@ -0,0 +1,72 @@
+<%= content_for(:title, "Search Results | MIT Libraries") %>
+
+<p class="id">
+  You requested record: <%= @id %>
```

Comment:
Yeah, with this change we don't need `id` to be an instance variable. I also discovered in this change that I'd used the old name for this field, so that was fixed as well.